Cargo.toml

```diff
@@ -10,6 +10,7 @@ homepage = "https://github.com/pingcap/grpc-rs"
 documentation = "https://docs.rs/grpcio"
 description = "The rust language implementation of gRPC, base on the gRPC c core library."
 categories = ["asynchronous", "network-programming"]
+autoexamples = false
```

Comment:
Seems to be an unrelated change?

Reply:
I change the pr title to better reflect the changes.

Reply:
It's possible to let it autodiscover, which requires changes to examples' structure. Set it to false is the simplest way to fix the warnings.